src/app/game-state/battle.service.ts

```diff
@@ -143,7 +143,15 @@ export class BattleService {
       for (let i = 0; i < enemyStack.quantity; i++){
         if (Math.random() < enemyStack.enemy.accuracy){
           let damage = enemyStack.enemy.attack;
-          damage = damage / (1 + this.characterService.characterState.defense * 0.1);
+          let defense = this.characterService.characterState.defense;
+          // with 20k max health currently, the curve has to be related to Death's damage so player has the chance to survive and oneshot Death.
```

Comment:
We're not planning to require a one-shot kill now, right? Want to update the comment?